docs/getting_started/openstack.rst

```diff
@@ -83,9 +115,9 @@ The playbooks get Username/Password information via environment variables:
 
    Your OpenStack password
 
-Before running any playbooks, run the following command to to pull in your
-username and password for Ansible to use, changing the file name and location to
-the location of your OpenStack RC file:
+Before running terraform or any playbooks, run the following command to to pull
+in your username and password for Ansible to use, changing the file name and
+location to the location of your OpenStack RC file:
 
 .. code-block:: shell
```

Comment:
Looks good to me, we really should have TF create the security group to reduce the number of steps, will ask a story to our backlog. @PeterLamar do you mind having a quick read too before i merge?

Matt